static_frame/core/store_zip.py

```diff
@@ -3,22 +3,67 @@
 import pickle
 from io import StringIO
 from io import BytesIO
-
+from concurrent.futures import ProcessPoolExecutor
 
 from static_frame.core.frame import Frame
 from static_frame.core.store import Store
 from static_frame.core.store import store_coherent_non_write
 from static_frame.core.store import store_coherent_write
 from static_frame.core.store import StoreConfigMap
+from static_frame.core.store import StoreConfig
 from static_frame.core.store import StoreConfigMapInitializer
 from static_frame.core.util import AnyCallable
 from static_frame.core.container_util import container_to_exporter_attr
+from static_frame.core.util import DtypesSpecifier
+
+
+FrameConstructor = tp.Callable[[tp.Any], Frame]
+
+
+class HashableConfigParams(tp.NamedTuple):
+    index_depth: int
+    index_name_depth_level: tp.Optional[tp.Union[int, tp.Iterable[int]]]
+    columns_depth: int
+    columns_name_depth_level: tp.Optional[tp.Union[int, tp.Iterable[int]]]
+    columns_select: tp.Optional[tp.Iterable[str]]
+    dtypes: DtypesSpecifier
+    consolidate_blocks: bool
+
+
+class BuildFrameParams(tp.NamedTuple):
```

Comment:
Maybe a better name for this is `FrameInitDeferred`, as we use this as a payload to defer initialization/construction of the `Frame`.